### PR TITLE
Fix for nil ptr deref and run gofmt

### PR DESCRIPTION
**Pull request by ishworgurung**

### CODE REVIEW

**Comment by ishworgurung** [comment]
I signed it!

**Comment by ishworgurung** [comment]
thanks @thaidn

**Comment by przydatek** [comment]
Thanks!

File: go/integration/gcpkms/gcp_kms_aead_test.go [hunk 1]
```diff
@@ -55,7 +55,10 @@ func setupKMS(t *testing.T) {
 	if err != nil {
 		t.Errorf("error setting up gcp client: %v", err)
 	}
-	_, err = g.LoadCredentials(credFile)
+	gcpClient, err := g.LoadCredentials(credFile)
+	if gcpClient == nil {
```
Comment (cuonglm) [inline go/integration/gcpkms/gcp_kms_aead_test.go]: We shouldn't check `gcpClient` here, instead, just revert and make `t.Errorf` at line 63 becomes `t.Fatalf`.

**Pull request merged**